### PR TITLE
docs: merge Threeport user docs into Qleet docs

### PR DESCRIPTION
**Pull request by lander2k2**

### CODE REVIEW

**Review by randalljohnson — APPROVED** [review]
Approved with a few minor suggestions

File: docs/img/threeport/APIDrivenDelivery.png
[the remark this answers is not in the document]
Comment (lander2k2) [reply docs/img/threeport/APIDrivenDelivery.png:1]: We're not.  That's in the current threeport docs that will soon be updated

File: docs/img/threeport/KubernetesFederationWithThreeport.png
[the remark this answers is not in the document]
Comment (lander2k2) [reply docs/img/threeport/KubernetesFederationWithThreeport.png:1]: Will be updated with threeport docs

File: docs/guides/add-aws-account.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+If you haven't already, [install qleetctl](/guides/install-qleetctl), the Qleet
+command line tool.
+
+## Add Account
+
+The following steps register and configure your AWS account in Qleet.  Once
+registered, you can spin up Kuberntes Runtimes and Workloads in your AWS account
+using Qleet.
+
+1. Set user environment variables by creating a file called `config-env-var`
```
[the remark this answers is not in the document]
Comment (lander2k2) [reply docs/guides/add-aws-account.md:14]: The user docs need to be built out further.  This PR is primarily for the threeport docs merge

File: docs/guides/add-aws-account.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Prerequisites
+
+If you haven't already, [install qleetctl](/guides/install-qleetctl), the Qleet
+command line tool.
+
+## Add Account
+
+The following steps register and configure your AWS account in Qleet.  Once
+registered, you can spin up Kuberntes Runtimes and Workloads in your AWS account
```
[the remark this answers is not in the document]
Comment (lander2k2) [reply docs/guides/add-aws-account.md:11]: Fixed this with a couple of other typos

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Getting Started
+
+Follow these steps to get up and running with Qleet:
+
+1. First, [install qleetctl](/guides/install-qleetctl), the Qleet command line tool.
+1. Next, create a new account with Qleet.  TODO
```
[the remark this answers is not in the document]
Comment (lander2k2) [reply docs/getting-started.md:6]: There's actually no way to do this currently.  I changed it to "Coming soon"

**Pull request merged**